client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-hosting-details-list.tsx

```diff
@@ -1,16 +1,19 @@
-import { trendingUp } from '@wordpress/icons';
+import { trendingUp, next } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
 import { defaultHostingDetails } from '../constants';
 import wordpressCwvtechReportJson from '../wordpress-cwvtech-report.json';
 import { useGetUpgradePlanSiteMetrics } from './use-get-upgrade-plan-site-metrics';
 
 export const useUpgradePlanHostingDetailsList = () => {
 	const translate = useTranslate();
-	const { showUpdatedSpeedMetrics, lcpPercentageDifference } = useGetUpgradePlanSiteMetrics();
-	const hostingDetails = [ ...defaultHostingDetails ];
+	const { lcpPercentageDifference, fidPercentageDifference } = useGetUpgradePlanSiteMetrics();
 
-	if ( showUpdatedSpeedMetrics ) {
-		const wordpressLCP = Math.round( 100 * wordpressCwvtechReportJson?.goodLCP );
+	let newDefaultHostingDetails = [ ...defaultHostingDetails ];
+	const newHostingDetails = [];
+
+	// LCP hosting details.
+	if ( lcpPercentageDifference > 0 ) {
+		const wordpressLCP = Math.floor( 100 * wordpressCwvtechReportJson?.goodLCP );
```

Comment:
Good point! But I ended up just removing the `?` because the file is imported locally from the static JSON. Make sense?

https://github.com/Automattic/wp-calypso/pull/90717/commits/be15cb1359f3b898525899e3d5984db6dba11763

Reply:
Yes perfect.